tests/unit/test_github_actions_hygiene.py

```
@@ -371,11 +374,45 @@ def test_actionlint_success_and_disabled_return_no_findings(self, tmp_path):
+        # 1. Disabled by config → "disabled".
+        disabled = GitHubActionsHygieneCheck({"run_actionlint": False}).run(
+            str(tmp_path)
+        )
+        assert disabled.status == CheckStatus.PASSED
+        assert "actionlint disabled" in disabled.output
+
+        # 2. Enabled + resolvable + clean run → "checked".
+        check = GitHubActionsHygieneCheck({"run_actionlint": True})
+        clean = MagicMock(returncode=0, output="")
+        with patch(find_tool, return_value="/usr/bin/actionlint"):
+            with patch.object(check._runner, "run", return_value=clean):
```

Ruff (0.15.17)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)